arm-resources/locks/2016-09-01/swagger/locks.json

```diff
@@ -0,0 +1,781 @@
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/ManagementLockListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-odata": "#/definitions/ManagementLockObject"
+      }
+    }
+  },
+  "definitions": {
+    "DeploymentExtendedFilter": {
+      "properties": {
+        "provisioningState": {
+          "type": "string",
+          "description": "Gets or sets the provisioning state."
+        }
+      },
+      "description": "Deployment filter."
+    },
+    "GenericResourceFilter": {
+      "properties": {
+        "resourceType": {
+          "type": "string",
+          "description": "Gets or sets the resource type."
+        },
+        "tagname": {
+          "type": "string",
+          "description": "Gets or sets the tag name."
+        },
+        "tagvalue": {
+          "type": "string",
+          "description": "Gets or sets the tag value."
+        }
+      },
+      "description": "Resource filter."
+    },
+    "ResourceGroupFilter": {
+      "properties": {
+        "tagName": {
+          "type": "string",
+          "description": "Gets or sets the tag name."
+        },
+        "tagValue": {
+          "type": "string",
+          "description": "Gets or sets the tag value."
+        }
+      },
+      "description": "Resource group filter."
+    },
+    "ManagementLockOwner": {
+      "properties": {
+        "applicationId": {
+          "type": "string",
+          "description": "Gets or sets the application Id of the management lock owner."
+        }
+      }
+    },
+    "ManagementLockProperties": {
+      "properties": {
+        "level": {
+          "type": "string",
+          "description": "Gets or sets the lock level of the management lock.",
+          "enum": [
+            "NotSpecified",
+            "CanNotDelete",
+            "ReadOnly"
+          ],
+          "x-ms-enum": {
+            "name": "LockLevel",
+            "modelAsString": true
+          }
+        },
+        "notes": {
+          "type": "string",
+          "description": "Gets or sets the notes of the management lock."
+        },
+        "owners": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagementLockOwner"
+          },
+          "description": "Gets or sets the owners of the management lock."
+        }
+      },
+      "description": "The management lock properties."
+    },
+    "ManagementLockObject": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ManagementLockProperties",
+          "description": "Gets or sets the properties of the lock."
+        },
+        "id": {
```

Comment:
Can we also identify all the properties that can't be set, and add a read-only to it? For reference: https://github.com/Azure/azure-rest-api-specs/pull/503/commits/c0f51dd4ead7fc3db9a81646ec15cbfd4793d229